kmip/pie/objects.py

```diff
@@ -657,7 +657,7 @@ class SymmetricKey(Key):
         'sqlite_autoincrement': True
     }
 
-    def __init__(self, algorithm, length, value, masks=None,
+    def __init__(self, algorithm, length, value, app_specific_info=None, masks=None,
```

Comment:
The position of the `app_specific_info` parameter is not consistent between the four init methods in #654 and #650.
It would be better to make it consistent, for example by making it the last parameter in all four methods.